content/vendor/snapshots/overview.md

```diff
@@ -16,3 +16,6 @@ In addition to the default functionality that Velero provides, KOTS provides a d
 
 Finally, KOTS exposes hooks that can be used to inject scripts to execute both [before and after a backup](/vendor/snapshots/configuring-backup) and [before and after a restore](/vendor/snapshots/configuring-restore).
 
+## Velero version compatibility
+
+Velero 1.2.0 is the currently supported version.
```

Comment:
Let's make this a table.  KOTS versions, Compatible Velero Versions.

Then we can reference it over time.